README.md

```diff
@@ -1,3 +1,3 @@
 # pymar2024
 Python test automation course
-Ekaterina Tsiunel HW-3-Git
```

Comment:
do not need to change this file in every HW

Reply:
I know) it was a one-time change